### DISCUSSION TITLE
The mod targets a wrong version of BepInEx?

### DISCUSSION THREAD

**Discussion by Hodurlein** [BudgetAirpods/InsanityRemastered#36] [General]
Like in the title, it says it targets the wrong version of BepInEx and a bunch of errors like this:
![Lethal_Company_viA6RpxuFx](https://github.com/BudgetAirpods/InsanityRemastered/assets/128432305/d5f2b8de-5044-4cc7-b2ae-0f1c275b0004) When I check Thunderstore's latest BepInEx, I have it latest, but the mod says there's a newer version?

**Comment by HossBigft** [comment]
I just checked, I have this one too but mod works. Maybe it's just warning?

**Comment by Justuuz** [comment]
tbf i have the same error, mod also not work for me anymore. I already tried to play with just Bepinex,Lc_api and insanity mod, somhow its wont work :(

**Reply by Justuuz** [reply]
Update: i fix it for me somehow

instead of 
"C:\Users\User\AppData\Roaming\Thunderstore Mod Manager\DataFolder\LethalCompany\profiles\Test\BepInEx\plugins\BudgetAirpods-InsanityRemastered"

I change it to 
"C:\Users\User\AppData\Roaming\Thunderstore Mod Manager\DataFolder\LethalCompany\profiles\Mod\BepInEx\plugins\InsanityRemastered"

i think the error cause was the wrong folder name. 
Instead of "BudgetAirpods-InsanityRemastered" i change it to "InsanityRemastered"

Now its totally fine
(i'm not a programmer or something, but it fixed my problem)

**Reply by Hodurlein** [reply]
Nice!
